packages/@glimmer/syntax/lib/parser/tokenizer-event-handlers.ts

```diff
@@ -463,10 +463,6 @@ export function preprocess(
 
   let program = new TokenizerEventHandlers(source, entityParser, mode).acceptTemplate(ast);
 
-  if (options.strictMode) {
-    program.blockParams = options.locals ?? [];
-  }
```

Comment:
Do we know if this something AST consumers actually rely on? (Specifically, the "block params" on a root level `Template` node has the `locals` name that were passed here) It seems like it would be incomplete/unreliable at best, and it wasn't used by any tests, so presumably we don't rely on it ourselves (we don't even read it)